app/services/background.py

```
@@ -577,3 +586,133 @@ async def _cleanup_expired_sessions(self):
+                    # Remove old log files
+                    old_logs = glob.glob(f"{log_dir}/*.log.*")
+                    old_logs.extend(glob.glob(f"{log_dir}/*-*.log"))
+                    
+                    for log_file in old_logs:
+                        try:
+                            file_path = Path(log_file)
+                            if file_path.stat().st_mtime < (time.time() - 7 * 24 * 3600):  # 7 days
+                                os.remove(log_file)
+                                cleanup_actions.append(f"Removed old log: {log_file}")
+                        except Exception:
+                            continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)